src/app/_components/MobileNavigation.tsx

```diff
@@ -0,0 +1,151 @@
+'use client'
+
+import { Fragment, useState, Dispatch, SetStateAction } from 'react'
+
+import Link from 'next/link'
+
+import { List, X, Icon as IconType } from '@phosphor-icons/react'
+import clsx from 'clsx'
+import { Route } from 'next'
+
+import { Icon } from '@/components/Icon'
+import { Logo } from '@/components/Logo'
+import { SlideOver } from '@/components/SlideOver'
+import { Social } from '@/components/Social'
+import { baseStyles } from '@/components/TextLink'
+
+import { PATHS } from '@/constants/paths'
+
+const getInvolvedItems = [PATHS.EVENTS, PATHS.GRANTS]
+const communityItems = [PATHS.ECOSYSTEM, PATHS.GOVERNANCE]
+
+type MobileLinkProps = {
+  label: string
+  path: Route
+  nested?: boolean
+  setOpen: Dispatch<SetStateAction<boolean>>
+}
+
+function MobileLink({ label, path, nested, setOpen }: MobileLinkProps) {
+  const touchTargetSpacing = 5
+  const touchTargetClass = `px-${touchTargetSpacing}`
+  const touchTargetOffsetClass = `-ml-${touchTargetSpacing}`
+
+  return (
+    <li
+      className={clsx(
+        nested ? 'border-l border-brand-400' : touchTargetOffsetClass,
+      )}
+    >
+      <Link
+        href={path}
+        className={clsx(
+          baseStyles,
+          touchTargetClass,
+          'inline-block px-5 py-2.5',
```

Comment:
Is `px-5` already the `touchTargetClass`?

```suggestion
          'inline-block py-2.5',
```

Reply:
Yes, it is; I forgot to add it!

About that, **I noticed that Tailwind classes coming from variables don't work**, as in they are present in the HTML but with no corresponding CSS, as shown in the screenshot below.

My guess is that because Tailwind strips out unused classes from the CSS file at build time, classes coming from variables are not counted. I `npm run build` the app locally and inspected the final CSS output, `-ml-5` is not part of it.

Tailwind probably only keeps the classes explicitly listed in `className`? I would have to research this more, maybe there's a config to make this work.

By the way, I also checked the negative margins in `<Social />`, `-m-2` coming from `touchTargetOffsetClass`, and it's not working either. `p-2` from `touchTargetClass` does work because we must use `p-2` explicitly somewhere else, so it gets picked up by Tailwind.

![CleanShot 2024-05-07 at 18 42 35](https://github.com/FilecoinFoundationWeb/filecoin-foundation/assets/20277959/3f83737c-a6ab-4042-b45d-1b68c8c15705)